src/item.h

```diff
@@ -714,8 +727,25 @@ class item : public visitable<item>
          */
         void calc_temp( const int temp, const float insulation, const time_duration &time );
 
+        /**
+         * Set temperature tags and counter according to a static counter
+         * @param counter Static counter representation of item temperature
```

Comment:
You do not need to write a separate sentence for every parameter. If the function description already explains what the parameter is used for, that's good and enough. This line adds no information and is just redundant clutter.